src/main/java/Command.java

```diff
@@ -0,0 +1,55 @@
+import java.util.function.BiConsumer;
+
+public enum Command {
+    DEADLINE("deadline", (input, taskLogs) -> {
+        Deadline deadline = new Deadline(input);
+        taskLogs.addTask(deadline);
+    }),
+    EVENT("event", (input, taskLogs) -> {
+        Event event = new Event(input);
+        taskLogs.addTask(event);
+    }),
+    TODO("todo", (input, taskLogs) -> {
+        ToDo todo = new ToDo(input);
+        taskLogs.addTask(todo);
+    }),
+    DELETE("delete", (input, taskLogs) ->
+            taskLogs.deleteTask(Integer.parseInt(input.trim().split(" ")[0]))),
+    MARK("mark", (input, taskLogs) ->
+            taskLogs.markTask(Integer.parseInt(input.trim().split(" ")[0]))),
+    UNMARK("unmark", (input, taskLogs) ->
+            taskLogs.unmarkTask(Integer.parseInt(input.trim().split(" ")[0]))),
+    FIND("find", (input, taskLogs) ->
+            taskLogs.printSingleTask(Integer.parseInt(input.trim().split(" ")[0]))),
+    LIST("list", (input, taskLogs) -> taskLogs.printTasks()),
+    HELP("help", (input, taskLogs) -> {
+        System.out.println("You sought help! "
+                + "As Dumbledore says, help will always be given to those who ask for it");
+        System.out.println(Help.getFullHelp());
+    }),
+    UNKNOWN("", (input, taskLogs) -> {
+        System.out.println("You typed an invalid command!");

```

Comment:
`UNKNOWN` does not specify what the invalid command was.

Include the invalid command in the error message to improve debugging.
```suggestion
        System.out.println("Invalid command: " + input);
```